.rubocop.yml

```diff
@@ -3,18 +3,10 @@ AllCops:
     - vendor/**/*
     - Guardfile
 
-AlignParameters:
```

Comment:
I believe the activemq cookbook has the current version of the .rubocop.yml file. In the near future the need for this will all go away.